src/Chainweb/Version.hs

```diff
@@ -637,6 +637,29 @@ pactBackCompat_v16 Mainnet01 h = h < 328000
 pactBackCompat_v16 Development h = h < 120
 pactBackCompat_v16 _ _ = False
 
+-- | If this is true the creation time of the current header is used for pact tx
+-- creation time and ttl validation.
+--
+-- Once the block height for triggering this on mainnet is in the past, the
```

Comment:
... but below, `Mainnet01` is always `True` so this para seems wrong

Reply:
The current value for `Mainnet01` is just a placeholder, since it's not yet decided when to do the decision.